### ISSUE TITLE
Bug: unexpected error occurs after choosing sass as css preprocessor

### ISSUE THREAD

**Issue by Rotten-LKZ** [fastjs-team/fastjs-cli#6]
**Describe the bug**
![Screenshots](https://user-images.githubusercontent.com/54099603/210130231-37262526-d8b5-4a09-bd15-3d89043726fd.png)
As shown in the picture

**To Reproduce**
1. Use `fastjs create` and choose sass as css preprocessor
2. `npm dev` to start this project
3. Then the error occures

**Expected behavior**
No error

**Screenshots**
![Screenshots](https://user-images.githubusercontent.com/54099603/210130231-37262526-d8b5-4a09-bd15-3d89043726fd.png)

**Additional context**
For sass (that is, the file suffix is `.sass`), the correct syntax should be like `Python`, without curly braces and semicolons, and written only by strict indentations.
```sass
$primary-color: #FF0000

body
  margin: 0
  color: $primary-color
```

And scss (that is, the file suffix is `.scss`) is a superset of css, which means essentially all valid CSS is valid SCSS as well.
```scss
$primary-color: #FF0000;

body {
  margin: 0;
  color: $primary-color;
}
```

*(See more: [Sass Official Documentation](https://sass-lang.com/documentation/syntax))*

Therefore, change `.css` to `.scss` or write a new set of `.sass` code separately

**Version**
![image](https://user-images.githubusercontent.com/54099603/210130289-5fd5a845-0b91-44f2-9c0a-4bfe9fd80172.png)
Node.js: v18.12.1
fastjs-cli: v2.4.3

**Comment by xiaodong2008** [comment]
Hi, this problem is in todo list now, thanks for your issue!

**Comment by xiaodong2008** [comment]
This problem has been fixed in commit f957c2ad20e4fbe6e139af87ca6116435fac7bdc
And it will be publish at release v2.4.4
